practice/src/index.html

```diff
@@ -3,10 +3,10 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <link rel="stylesheet" href="../src/css/index.css">
+  <link rel="stylesheet" href="../src/css/main.css">
```

Comment:
Should add a Favicon in next MR. https://www.w3schools.com/html/html_favicon.asp